docs/overview/renderers/nema_gfx.rst

```diff
@@ -54,6 +54,27 @@ no locking is done). It may conflict with existing definitions
 if you have an existing Nema HAL implementation. You may
 simply be able to remove yours.
 
+TSC Images
+**********
+
+TSC (ThinkSillicon Compression) images can be drawn by this renderer. The
+TSC 4/6/6A/12/12A color formats are part of :cpp:type:`lv_color_format_t`.
+All other renderers will ignore images with these color formats.
+Define an image descriptor variable with the corresponding
+TSC color format and the GPU will be able to draw it directly.
+Stride does not need to be specified because it will be computed by the
+renderer.
+
+.. code-block:: c
+
+    const lv_image_dsc_t img_demo_widgets_avatar_tsc6a = {
+        .header.cf = LV_COLOR_FORMAT_NEMA_TSC6A,
+        .header.w = 144,
+        .header.h = 144,
+        .data = img_demo_widgets_avatar_tsc6a_map,
+        .data_size = sizeof(img_demo_widgets_avatar_tsc6a_map),
+    };
```

Comment:
How we generate the converted images?

Reply:
It's not clear to me. Think Silicon has a tool I believe. cc @MarkIoanAmat

Reply:
Yes we have a tool for such tasks named NEMA Pix-Presso, it is free for download here:
https://www.think-silicon.com/nema-pix-presso
@XuNeo I created this image in 2 steps and that's what anyone who wants to use are proprietary format should do, at least for the time being.

1. Use pix-presso to convert the image to the desired format (TSC4/6/12 etc) and save it as a header file
2. Use the data generated to populate a lv_image_dsc_t struct and save it to a .c file (by hand for now)